addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -3388,7 +3389,8 @@ export class Wysiwyg extends Component {
                     'image/jpeg': canvas.toDataURL('image/jpeg', 0.75).split(',')[1],
                 };
                 if (size !== originalSize) {
-                    altData[size]['image/webp'] = canvas.toDataURL('image/webp', 0.75).split(',')[1];
+                    const { dataURL, mimetype: outputMimetype } = convertCanvasToDataURL(canvas, 'image/webp', 0.75);
+                    altData[size][outputMimetype] = dataURL.split(",")[1];
```

Comment:
Here, we could probably just skip this if `!canExportCanvasAsWebp()` as we already have a jpeg alternative just above.